react/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "scripts": {
     "pretest": "yarn",
-    "test": "vtex-test-tools test"
+    "test": "vtex-test-tools test",
+    "lint": "eslint **.js"
```

Comment:
I tested and could not make this work, use:
`"lint": "eslint --ext js,jsx,ts,tsx .",`